tools/clang/test/CodeGenSPIRV/spv.enable-maximal-reconvergence.hlsl

```diff
@@ -0,0 +1,11 @@
+// RUN: %dxc -T cs_6_0 -E main -fspv-enable-maximal-reconvergence %s -spirv | FileCheck %s -check-prefix=CHECK-ENABLED
+// RUN: %dxc -T cs_6_0 -E main %s -spirv | FileCheck %s -check-prefix=CHECK-DISABLED
+
+// CHECK-ENABLED:            OpExtension "SPV_KHR_maximal_reconvergence"
+// CHECK-DISABLED-NOT:       OpExtension "SPV_KHR_maximal_reconvergence"
+// CHECK:                    OpEntryPoint GLCompute %main "main"
```

Comment:
Does this get checked? I through that "CHECK" was the default, but was not checked if you specify a prefix.

Reply:
Good catch. The command is not additive. re-added the CHECK prefix too to have the shared one. Checked by modifying the lines, and the 2 CHECKS are checked, flags are not exclusive.